_posts/2021-08-19-Spencer-Showcase.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Scaling virtual screening to ultra-large virtual chemical libraries" 
+date: 2021-08-19T12:00:00+00:00
+excerpt: Kicking off last week’s OSG User School Showcase, Spencer Erikson, a researcher at the University of Wisconsin-Madison’s Carbone Cancer Center, described how high throughput computing (HTC) has made his work in early-stage drug discovery infinitely more scalable.
```

Comment:
Ericksen

Reply:
@josiewatkins

Reply:
Fixed and submitted a new PR, thanks for catching Tim